core/src/main/java/org/mule/model/resolvers/MethodHeaderPropertyEntryPointResolver.java

```diff
@@ -56,8 +55,8 @@ public InvocationResult invoke(Object component, MuleEventContext context) throw
 
         //TODO MULE-4953 I don't think the VM transport if managing invocation properties correctly, or maybe it is and this
```

Comment:
Not related to your comment, but MULE-4953 is fixed and we don't have VM transport anymore. Can you remove it?